src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -732,6 +735,13 @@ private void visitType(JSType typeToVisit) {
       // See also JsdocToEs6TypedConverter in the Closure code base. This code is implementing the
       // same algorithm starting from JSType nodes (as opposed to JSDocInfo), and directly
       // generating textual output. Otherwise both algorithms should produce the same output.
+      if (isPrivate(typeToVisit.getJSDocInfo())) {
```

Comment:
Always find it crazy that we need to dance around JSDocInfo to get the visibility of a symbol. (no action required).